src/bootstrap/flags.rs

```diff
@@ -199,6 +200,13 @@ To learn more about a subcommand, run `./x.py <subcommand> -h`"
                     "mode describing what file the actual ui output will be compared to",
                     "COMPARE MODE",
                 );
+                opts.optopt(
+                    "",
+                    "force",
+                    "force testing to stop at a certain stage;\
```

Comment:
I would not use "stage" as that terminology has different meaning in bootstrap.

Reply:
What is a good name for `check`, `build` and `run` collectively (considering that they are `Ord`ered)?
Level?

Reply:
I'll go with "level"; honestly I didn't like "stage" very much but couldn't find anything better.

Reply:
In https://github.com/rust-lang/rust/pull/61778 I ended up with `PassMode`, which can be shortened to `--pass` on the command line.

`// check-pass` <-> `--pass check` seems pretty reasonable to me.